### ISSUE TITLE
Lebensohl error

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#112]
04E701F900A520D2AB5F263B4A37
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/5f46c45b-9b5a-4837-8552-a9755f2da526)

**Comment by EdwardPiwowar** [comment]
No, maybe 3C is too weak but 3NT is game try.

**Comment by ThorvaldAagaard** [comment]
Yes, it is to weak as a gametry - with a 6-card suit and 2 of 3 honors it could be OK
